datamodel.graphql

```diff
@@ -112,6 +112,20 @@ type ClassLevel {
     classRooms: [Classroom]
 }
 
+type TimeTable {
+    id: ID! @id
```

Comment:
Add field to indicate if it's allDay

Reply:
There is an enum for that. The timetable type

Reply:
Is that not for checking whether it's single day or recurring? What I mean is indicating whether an event will take place the whole day(morning to evening)